File: 09/CS/CustomControls/09CustomControls/Client/UserCode/RichTextControl908.cs
using System;
using System.Linq;
using System.IO;
using System.IO.IsolatedStorage;
using System.Collections.Generic;
using Microsoft.LightSwitch;
using Microsoft.LightSwitch.Framework.Client;
using Microsoft.LightSwitch.Presentation;
using Microsoft.LightSwitch.Presentation.Extensions;

namespace LightSwitchApplication
{
    public partial class RichTextControl908
    {
        partial void RichTextControl908_InitializeDataWorkspace(global::System.Collections.Generic.List<global::Microsoft.LightSwitch.IDataService> saveChangesTo)
        {
            // Write your code here.
            this.ProductProperty = new Product();
        }

        partial void RichTextControl908_Saved()
        {
            // Write your code here.
            this.Close(false);
            Application.Current.ShowDefaultScreen(this.ProductProperty);
        }

        partial void RichTextControl908_Activated()
        {
            // Write your code here.

        }
    }
}

File: 09/VB/09CustomControls/Client/UserCode/RichTextControl908.vb

Namespace LightSwitchApplication

    Public Class RichTextControl908

        Private Sub RichTextControl908_InitializeDataWorkspace(ByVal saveChangesTo As Global.System.Collections.Generic.List(Of Global.Microsoft.LightSwitch.IDataService))
            ' Write your code here.
            Me.ProductProperty = New Product()
        End Sub

        Private Sub RichTextControl908_Saved()
            ' Write your code here.
            Me.Close(False)
            Application.Current.ShowDefaultScreen(Me.ProductProperty)
        End Sub

        Private Sub RichTextControl908_Activated()
            ' Write your code here.

            Dim rt As IContentItemProxy = Me.FindControl("RichTextDescription")
            rt.SetBinding(CentralControls.RichTextEditorInternal.TextProperty, "Value", Windows.Data.BindingMode.TwoWay)

        End Sub
    End Class

End Namespace